lib/src/file_picker_macos.dart

```diff
@@ -136,8 +136,6 @@ class FilePickerMacOS extends FilePicker {
         return '"avi", "flv", "m4v", "mkv", "mov", "mp4", "mpeg", "webm", "wmv", "bmp", "gif", "jpeg", "jpg", "png"';
       case FileType.video:
         return '"avi", "flv", "mkv", "mov", "mp4", "m4v", "mpeg", "webm", "wmv"';
-      default:
```

Comment:
This is reported by https://dart.dev/tools/diagnostic-messages?utm_source=dartdev&utm_medium=redir&utm_id=diagcode&utm_content=unreachable_switch_default#unreachable_switch_default